src/libcore/iter.rs

```diff
@@ -607,20 +607,28 @@ pub trait IteratorExt: Iterator + Sized {
     /// Performs a fold operation over the entire iterator, returning the
     /// eventual state at the end of the iteration.
     ///
+    /// # Arguments
```

Comment:
We don't currently have any conventions for using `Arguments` as a top-level thing.